lib/spack/docs/configuration.rst

```diff
@@ -78,6 +78,13 @@ are six configuration scopes. From lowest to highest:
    If multiple scopes are listed on the command line, they are ordered
```

Comment:
Can you update the number of configuration scopes at the top of this section from 6 to 7? Or just remove the number...